pkg/deployment/deployment.go

```diff
@@ -113,6 +116,17 @@ func (d *Deployer) Deploy(services []string) (*ctrl.Result, error) {
 		}
 
 		log.Info(fmt.Sprintf("Condition %s ready", readyCondition))
+
+		// (TODO) Only considers the container image values from the Version
+		// for the time being. Can be expanded later to look at the actual
+		// values used from the inventory, etc.
+		if d.Version != nil {
+			vContainerImages := reflect.ValueOf(d.Version.Status.ContainerImages)
+			for _, cif := range foundService.Spec.ContainerImageFields {
+				d.Deployment.Status.ContainerImages[cif] = reflect.Indirect(vContainerImages.FieldByName(cif)).String()
```

Comment:
TODO: It's not guaranteed that the container image value read here is what was actually used. We haven't considered ansible group/host/extra vars. We might need to check that.